src/main/kotlin/lotto/Lotto.kt

```diff
@@ -0,0 +1,47 @@
+package lotto
+
+class Lotto {
+    fun buyTickets(pay: Int): Int {
+        return pay / TICKETPRICE
+    }
+
+    companion object {
+
+        const val TICKETPRICE = 1000
+    }
+
+    fun tickets(totalTickets: Int): MutableList<MutableList<Any>> {
+
+        val sampleTickets = mutableListOf(mutableListOf<Any>())
```

Comment:
`mutableListOf<Any>()`를 넣어 초기화한 뒤에 아래에서 removeAt(0)을 호출하는 것 보다 아예 값을 넣지 않는게 어떨까요

```suggestion
        val sampleTickets = mutableListOf<MutableList<Int>>()
```

여럿이 같이 개발을 진행할 때 이렇게 작성하시면 같이 개발을 진행하는 다른 사람들이 의도를 알아차리기 힘들어질 수 있어요.